game/scripts/npc/heroes/slark.txt

```diff
@@ -0,0 +1,19 @@
+"DOTAHeroes"
+{
+  //=================================================================================================================
+  // HERO: Slark
+  //=================================================================================================================
+  "npc_dota_hero_slark"
+  {
+    // Abilities
+    //-------------------------------------------------------------------------------------------------------------
+    "Ability10"   "special_bonus_strength_20" // replaces special_bonus_strength_10
+    "Ability11"   "special_bonus_agility_15" // "special_bonus_agility_6"
```

Comment:
Revert a buff to agility and attack speed talent, I made echo sabre buyable at the start and buffed its dmg.